nolus/assetlist.json

```diff
@@ -787,6 +787,156 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/celestia/images/celestia.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/celestia/images/celestia.svg"
       }
+    },
+    {
+      "description": "STRD transferred from the Osmosis mainnet that is supported on the Nolus chain",
+      "denom_units": [
+        {
+          "denom": "ibc/04CA9067228BB51F1C39A506DA00DF07E1496D8308DD21E8EF66AD6169FA722B",
+          "exponent": 0,
+          "aliases": [
+            "ustrd"
+          ]
+        },
+        {
+          "denom": "strd",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/04CA9067228BB51F1C39A506DA00DF07E1496D8308DD21E8EF66AD6169FA722B",
+      "name": "STRD",
+      "display": "strd",
+      "symbol": "STRD",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "ibc/A8CA5EE328FA10C9519DF6057DA1F69682D28F7D0F5CCC7ECB72E3DCA2D157A4",
```

Comment:
Please register this asset on the counterparty chain. (i.e., add STRD to Osmosis' assetlist)

Reply:
Done